created Traditional Chinese translation and updated Chinese translations to current English source

fire12324344 opened the pull request.

fire12324344 committed:
Update Chinese Translation to new English source. 

work in progress

fire12324344 committed:
Patch

Fixed "only show secure messages" to not sound dumb

fire12324344 committed:
Created Traditional Chinese translation

made Traditional Chinese translation which is basically the exact same as Simplified but with different hanzi and slightly different punctuation.

fire12324344 commented:
这样翻没有问题吧 @Taskeren

fire12324344 committed:
Update zh_tw.json

ignore this
did not accidentally space after every period 
definitely

dongffan commented:
![ScreenShot 2022-07-28 095612](https://user-images.githubusercontent.com/86098324/181403567-727d9035-18e7-4ce5-877d-e0d258e089ce.png) 😕

fire12324344 committed:
Changed "Only Show Secure Chat" to official translation

turns out you can access new chat settings on any world and not just servers.

fire12324344 committed:
Update zh_tw.json

uses official translation for only show secure chat which is twice as long and unnecessarily formal but, consistency Ig.

Taskeren commented on src/main/resources/assets/nochatreports/lang/zh_cn.json:
```diff
@@ -2,8 +2,8 @@
 	"disconnect.nochatreports.client": "此服务器没有安装 No Chat Reports，但你已将其配置为要求在服务器上安装。",
-	"gui.nochatreports.status_secure": "此服务器安装了 No Chat Reports 或其他已知的方法避免你的聊天被举报。你可以安全地聊天。",
-	"gui.nochatreports.status_unintrusive": "此服务器不能防止聊天举报，但允许发送未签名的消息。技术上来讲，你还是有可能被举报，但不太可能被视为有效的证据，同时也不太可能因此被处理。",
+	"gui.nochatreports.status_secure": "此服务器安装了 No Chat Reports 或其他已知的方法避免消息被举报。这也会避免 \"仅显示安全的聊天\" 挡你的消息。你可以安全地聊天。",
```
“挡” 换成 “屏蔽” 或者 “阻挡” 等其他的两字动词会更好一点。

Taskeren commented on src/main/resources/assets/nochatreports/lang/zh_cn.json:
```diff
@@ -2,8 +2,8 @@
 	"gui.nochatreport.secureChat": "此选项无法与 No Chat Reports 一起使用，因为没有可用于验证的签名。",
 	"gui.nochatreport.noReporting": "聊天举报在此服务器上被禁用，由 No Chat Reports 提供。",
 	"disconnect.nochatreports.client": "此服务器没有安装 No Chat Reports，但你已将其配置为要求在服务器上安装。",
-	"gui.nochatreports.status_secure": "此服务器安装了 No Chat Reports 或其他已知的方法避免你的聊天被举报。你可以安全地聊天。",
```
5-7 行的内容有新增。

Taskeren commented on src/main/resources/assets/nochatreports/lang/zh_tw.json:
```diff
@@ -0,0 +1,13 @@
+	"gui.nochatreports.status_insecure": "此伺服器要求所有玩家的消息簽名，你也同意在這設置下繼續遊戲。你所發送的任何消息都可能成為你被舉報時潛在的違規證據。你可以在 NoChatReports.json 中手動移除此伺服器，以再次獲得相同的警告。",
+	"gui.nochatreports.status_unknown": "No Chat Reports 無法檢測此伺服器的安全性。這不應該發生，請將此錯誤反饋給模組作者。",
+	"gui.nochatreports.unsafe_server.header": "警告：不安全的伺服器",
+	"gui.nochatreports.unsafe_server.contents": "此伺服器要求所有消息簽名，這意味著它們可以被舉報給 Mojang。可能是伺服器默認開啟了 「enforce-secure-profile」 選項。你可以選擇繼續遊玩，但 No Chat Reports 不能提供任何保護。",
```
我漏翻了，怎么你也漏翻了？是不是少了点东西啊。🙄

fire12324344 replied:
> 我漏翻了，怎么你也漏翻了？是不是少了点东西啊。🙄

line 10 很难翻 😅

Taskeren reviewed (changes requested):
Much better in general.
Maybe need to change the marked details.

fire12324344 committed:
Update zh_tw.json

lots of stuff

fire12324344 committed:
Update zh_tw.json

reverting line 10 for now
